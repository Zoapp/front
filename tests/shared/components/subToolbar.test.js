import React from "react";
import renderer from "react-test-renderer";
import SubToolbar from "@shared/components/subToolbar";

describe("SubToolbar", () => {
  it("renders correctly", () => {
    const component = renderer.create(
      <SubToolbar
        titleName="some-name"
      />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
