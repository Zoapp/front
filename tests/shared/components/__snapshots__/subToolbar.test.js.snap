// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubToolbar renders correctly 1`] = `
<div
  className="mrb-panel-header"
>
  <div
    className="mrb-panel-header-title"
    style={
      Object {
        "marginLeft": "16px",
      }
    }
  >
    some-name
  </div>
  <div
    className="mrb-panel-header__actions"
  >
    <div />
    <div />
  </div>
</div>
`;
